cypress/e2e/gui/mudarTema.cy.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
+
+describe('Validar Cabeçalho e Mudar o Tema', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000/')
+
+    });
+
+    it('Tema escuro para claro', () => {
+
+        cy.get('#theme-toggle-button')
+            .should('be.visible')
+            .click()
+
+        cy.get('h1').should('exist') // Verifica que o h1 existe dentro do cabeçalho
+            .and('be.visible') // Verifica que o h1 está visível
+            .and('contain.text', 'EngageSphere') // Verifica que o h1 contém o texto esperado (substitua pelo texto correto)
```

Comment:
Perceba que no meu exemplo (dois comentários atrás) fiz o mesmo com apenas uma linha de código.

Ou seja, não há motivo para fazer um `cy.get().should('contains', ...)`

Você pode fazer um único `cy.contains()`, onde seu primeiro argumento é um seletore CSS (ex.: a tag HTML do elemento) e o segundo seu conteúdo (seu texto).

Ex.: `cy.contains('h1', 'EngageSphere')`

Reply:
obrigada pela dica